src/app/Processor/types.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseAction } from "../../../openapi/generated";
```

Comment:
In my opinion, when a type/interface deserves a dedicated file, it is better to put it under `/src/types/`, grouping types by domain. A file named `types.ts` seems to me like a container of ungrouped things.

Reply:
Well my idea was to put a type file within the "domain" folder for types of that domain.

But I don't have strong opinions about it. I see you already created `/src/types/Instance.ts` so I've followed the same approach now.